### ISSUE TITLE
Add script to run all untested gallery examples locally

### ISSUE THREAD

**Issue by jni** [napari/napari#6393]
## 🧰 Task

As @GenevieveBuckley has noted, some of our examples are broken. I haven't compared the list thoroughly but I expect that those that are excluded from testing [here](https://github.com/napari/napari/blob/1b510bab020aae485565f000ddab842ab17ef608/napari/_tests/test_examples.py#L23-L32) are most likely to be broken. Since not every script can be tested, it would be good to have a local script that simply runs each of those examples in sequence, so that we can manually test them periodically.

Or if anyone has a better idea... 😂

**Comment by jni** [comment]
(We can use the `sphinx-gallery` default of anything with an underscore in it, and then the script can be as simple as:

```sh
for file in ../examples/*_.py; do
    python $file
done
```
)

**Comment by Czaki** [comment]
@jni Your proposition is not a solution. We perform such checks as part of the workflow. The problem is that such an approach does not allow to interact with the interface. For example last broken example requires clicking "Run" button to trigger calculations that leads to exceptions. 

So, it will require attaching some script to each example that will interact with its elements.

**Comment by Czaki** [comment]
Yes. But we could start from add comments with instruction how to test/play with given example. Then there wił be possible to slowly automatize check of some simpler examples.

**Comment by GenevieveBuckley** [comment]
> For example last broken example requires clicking "Run" button to trigger calculations that leads to exceptions.

With magicgui, you can just call the function directly - that's equivalent to clicking the "Run" button in the GUI.

**Comment by Czaki** [comment]
until it uses threading
